bundles/org.openhab.binding.wolfsmartset/README.md

```diff
@@ -0,0 +1,45 @@
+# Wolf Smartset Binding
```

Comment:
The README has to describe how to use the binding with the device. This isn't here the case. It's also mostly in german which is not readable for a great part of the community. Please rewrite it.

Reply:
correct, the readme is currently just a placeholder.
I wanted to get some initial thoughts on my code to refine it.